docs/source/get_involved/code_of_conduct.rst

```diff
@@ -168,15 +229,14 @@ GitHub.
 3.1 Contact points
 ~~~~~~~~~~~~~~~~~~
 
-If you feel able to, please contact Dr Franz Király by email at
-f.kiraly@ucl.ac.uk.
+Please report CoC incidents via the [reporting tool in the issue tracker](https://github.com/alan-turing-institute/sktime/issues/new/choose).
```

Comment:
I was curious about how the reporting tool worked, so opened the link and the page doesn't seem to exist?  If we are still in the process of building it - I had some suggestions/ideas as to what form this could take:
 - what if instead of just making a CoC incident issue tracker we use an (optionally anonymous) feedback form, which contains within it the functionality for anyone to report a CoC incident (which would of course then be flagged for more stringent triage)?  Emphasis on it being a *feedback form* over a CoC incident reporting tool as something I have seen work well in communities I have been a part of is the form can also serve to collate issues/ideas/suggestions from the community that can be addressed at the next community meeting.  It also makes it open to all sorts of feedback (ie positive feedback or new ideas) instead of just CoC complaints!  Of course there can be drawbacks to anonymity, but in my experience I have found that it sometimes lowers the bar to entry and can encourage more feedback from the community.
 - We could implement such a form via Google Forms - which could then be shared with all core contributors/a outside third party for review. 
 - If concerned about misuse we could remove the option for anonymity for a formal CoC complaint, but this could potentially keep the door open for people to provide feedback on sktime community and experience without needing to escalate to a CoC complaint? 
 
 If interested I can provide a mock-up of what a form could look like on Google Forms?

Reply:
>I was curious about how the reporting tool worked, so opened the link and the page doesn't seem to exist?

It should exist! Maybe this got destroyed somehow when we migrated documentation and no one checked 😢 
I think as a take-away, we should add, on the responsibility list of the CoC committee to perform regular checks of the reporting workflows and ensure they are kept up to date. Perhaps asking for a demonstration of a clickthrough every 6 months, to the CC.

> what if instead of just making a CoC incident issue tracker we use an (optionally anonymous) feedback form, which contains within it the functionality for anyone to report a CoC incident (which would of course then be flagged for more stringent triage)? Emphasis on it being a feedback form

Excellent idea! I am in favour

> If concerned about misuse

I think, in the balance of all things, misuse is less likely than genuine reports that want to remain anonymous. Even if more likely, I personally would weigh the trade-off strongly in favour of the latter.

> If interested I can provide a mock-up of what a form could look like on Google Forms?

Useful, in faour! What do you think, @aiwalter, @Lovkush-A ?

Where did the old reporting form end up being? @mloning, do you still know where it was/should be?

Reply:
@martinagvilas, do you know where the CoC reporting form is? See above, it seems the reporting tool link is now broken.

Reply:
[Here is an example](https://docs.google.com/forms/d/e/1FAIpQLSdaIpWJ2bAnkBBuYwCrAyzixHJ_n23IxkcdB6zJvFvW2_u_Ug/viewform?usp=sf_link) of what the form could look like - very short/simple.
  I went ahead and made both @fkiraly and @Lovkush-A editors, but happy to share that with anyone else who is interested as well.  I was thinking we could add the form to the end of all official correspondences (ie to the signature of the emails from the sktime toolbox?) and could also setup a slack bot that invites people to share feedback once a month? (or maybe on a similar frequency as the core dev meetings?).  In my experience such forms don't necessarily get used too heavily, but its nice to have them around! :)

Reply:
I like the short concise nature of form. Main comment is to make clear that there are two possible places the feedback can be used:

- For CoC issues, then only seen by CoC group.
- For everything else, discussed by core devs.

Reply:
generally, every form needs to say the purpose if is used for and the entity/recipient that will have a view of contents

Reply:
In that case I think what makes the most sense is:
- the results of this form are only visible to the CoC committee.  
- they are responsible for sharing all responses that don't indicate that they are CoC violations with the rest of the core dev team during meetings? 

I will add those changes in addition to a more specific explanation of who will read the responses/in what context to the above form shortly! :)

Reply:
yes, that makes sense. From a technical standpoint, I suppose we then need a gmail email that is shared by CoC committee? Or did you imagine this differently? 

Not sure about gmail workings, but there needs to be a "main" gmail acct for all google forms, right? Or wrong?

Reply:
Yea!  I went ahead and make you the owner Franz (since I am not on the CoC committee don't really think it makes sense for me to have access to this form once it goes live!).  

From a technical perspective I agree the best way to do it could be to have a shared mailing list (gmail based) that is for the CoC committee.  I think this would make it easier to add/remove people as needed.  and I imagine there would be a way to set up regular digests/updates to people who are subscribed when the form is updated?

Reply:
opened an issue here to set this up "properly".
https://github.com/sktime/sktime/issues/3664

Apologies for leaving this for a while...
I started my new job on Aug 3 and through that my capacity (and attention) got much reduced.